### ISSUE TITLE
Can't persist custom resolution on BigSur

### ISSUE THREAD

**Issue by mbeaudru** [usr-sse2/RDM#22]
After updating to BigSur, I tried to re-apply my custom HiDPI resolution but RDM doesn't persist the resolution change.

I disabled SIP following those instructions:

```
Click the Apple symbol in the Menu bar.
Click Restart…
Hold down Command-R to reboot into Recovery Mode.
Click Utilities.
Select Terminal.
Type csrutil disable.
Press Return or Enter on your keyboard.
Click the Apple symbol in the Menu bar.
Click Restart
```

And tried to add a custom resolution, then restart to persist the changes.

Usually (I did this on several machines) it works at this point, but this time after a restart I don't see the custom resolution with the ⚡ .

Do you know how to fix this ? Thank you ❤️

**Comment by ruotianluo** [comment]
I used https://github.com/xzhih/one-key-hidpi add the custom relotions, and was able to change the resolution using RDM.

**Comment by usr-sse2** [comment]
On Big Sur it should write to `/Library` instead of `/System/Library`. The code is already in master, but not ready for release. The next release will integrate the capabilities of RDM and one-key-hidpi. In fact, one-key-hidpi does many unnecessary things because it just copies some values from Apple plists without understanding what they mean.

**Comment by ruotianluo** [comment]
@usr-sse2 Thanks, can wait to see that. I tried to build with the default branch but failed.

**Comment by jnooree** [comment]
@ruotianluo You will need xcode command line tools for build. I haven't tested on Big Sur yet, but I don't see any OS dependencies so the scripts might be working.

**Comment by orospakr** [comment]
@usr-sse2 are you blocked on the issue of getting writable access to the filesystem underlying /System/Library on Big Sur?

**Comment by usr-sse2** [comment]
> @usr-sse2 are you blocked on the issue of getting writable access to the filesystem underlying /System/Library on Big Sur?

No, I know that files should go to /Library. I've done it manually but have no time now, and there are also other features planned for the release and partially implemented but not ready.

**Comment by orospakr** [comment]
Yeah, that seems to be the case.  I found a long thread on the Apple forums, and apparently provided you copy over the rest of the Displays bundle from /System/Library into /Library before adding your own custom override plist, it should theoretically work. No SIP required. https://developer.apple.com/forums/thread/649832

fwiw I couldn't get this to work myself though, but this may be because as I've now I've come to realize my HiDPI (lack thereof) issues are likely because of pretty seriously bogus EDID information coming from my 4K LG TV which have caused me problems with other devices too (claiming 1080p only, with a ProductID of `1` 🤪 ). I'm now going to try solving it with an EDID hardware emulator dongle now in the mail from Amazon.

Also the Mac in question is an Apple Silicon M1, so who knows 🤷‍♂️  (although the master branch of RDM does build and run on it haha)
